tests/test_opds.py

```diff
@@ -508,7 +508,7 @@ def test_acquisition_feed_includes_license_source(self):
         expect = '<bibframe:distribution bibframe:ProviderName="%s"/>' % (
             gutenberg.name
         )
-        assert (1, unicode(feed).count(expect))
+        assert 1 == unicode(feed).count(expect)
```

Comment:
Unrelated change -- the assertion was written wrong and it caused a warning while running tests which I happened to notice.